src/Directory.build.props

```diff
@@ -60,11 +60,8 @@
 
   <ItemGroup>	
     <PackageReference Include="Nerdbank.GitVersioning" Version="3.4.231" PrivateAssets="all" />	
-  </ItemGroup>
 
-  <ItemGroup>
     <PackageReference Include="stylecop.analyzers" Version="1.2.0-beta.354" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3" PrivateAssets="all" />
```

Comment:
Believe we still this for the non 6 based target frameworks

Reply:
V6 is automatically installed with net6 sdk, a warning is given for each target if V5 is used, so will leave as V6